app/scripts/shell/dimAppCtrl.controller.js

```diff
@@ -19,8 +19,11 @@
     });
 
     $scope.$watch('app.settings.showNewAnimation', function(showAnimation) {
-      var animation = showAnimation ? 'new_overlay 5s infinite' : 'initial';
-      document.querySelector('html').style.setProperty("--overlay-animation", animation);
+      if (showAnimation) {
+        $('body').addClass('new-item-animated');
+      } else {
+        $('body').removeClass('new-item-animated');
+      }
```

Comment:
Check out how this is done elsewhere. You don't need a watch, and you don't need add/remove class. Look in `index.html` and you can see where we bind some `ng-class` properties to `app.settings`.